_exercises/advanced/arrays/1-method-chaining.js

```diff
@@ -12,12 +12,19 @@ const episodes = require('../../../data/south-park')._embedded.episodes;
 
  */
 
+  const getAllEpisodesBySeason = ( season ) => episodes.filter(function( episode) {
```

Comment:
You don't have to write **function (**... just use a lambda

(episode) => episode.season === season;